aspnetcore/performance/rate-limit.md

```diff
@@ -127,6 +129,36 @@ The following code uses the concurrency limiter:
 
 :::code language="csharp" source="~/../AspNetCore.Docs.Samples/fundamentals/middleware/rate-limit/WebRateLimitAuth/Program.cs" id="snippet_concur":::
 
+## `EnableRateLimiting` and `DisableRateLimiting` attributes
```

Comment:
This section requires review and signoff.

Reply:
It's worth calling out that `DisableRateLimiting` disables both named policies _and_ the global limiter (if present)